<img width="2005" height="1137" alt="Graphical_abstract_epigenetic_paper_v2" src="https://github.com/user-attachments/assets/4a90581a-987e-486a-83e5-8123b8b04d7a" />

Accurate age estimation is key to understanding life history, population ecology, and effective management of valuable commercial and ecologically relevant species. While DNA methylation-based epigenetic clocks are well developed in mammals, their application in fish is rare. Here, we explore the potential of ultra-conserved elements (UCEs) as age predictor markers in albacore tuna (Thunnus alalunga), a worldwide distributed and commercially valuable teleost. We produced a de novo reference genome of the albacore tuna in order to facilitate accurate mapping of bisulfite sequencing reads and identify conserved genomic regions. From muscle tissue samples spanning the 0.03 to 17.69 year age range, we profiled methylation at UCEs and selected ones whose levels of methylation are strongly correlated with age. By selecting an optimized subset of these regions, we constructed an epigenetic clock that, in a median absolute error sense, accurately predicts age to less than one year. This demonstrates that UCEs represent a conserved set of age markers in teleosts and supports their potential for constructing shared epigenetic clocks between teleost species. To complement our clock construction based on penalized regression, we additionally conducted the first epigenome-wide association study (EWAS) of age in a fish species, providing a broader characterization of age-associated methylation patterns beyond the CpGs retained in the predictive model. Our strategy offers a non-lethal, real-world solution to age estimation from small tissue biopsies rather than otolith sampling and reduces the resources required to develop species-specific clocks.
